src/corelibs/U2Formats/src/PDBFormat.cpp

```diff
@@ -606,6 +609,15 @@ void PDBFormat::PDBParser::createMolecule(char chainIdentifier, BioStruct3D &bio
 void PDBFormat::PDBParser::updateResidueIndexes(BioStruct3D & /*biostruc*/) {
 }
 
+int PDBFormat::PDBParser::endOfNameInd(const QString &specification) {
+    int index = QRegExp(";\\s*$").indexIn(specification);
```

Comment:
Could you please re-use regex instance and avoid new instance creation on every method call?

Reply:
The best way is to create a `static const` variable